docs/getting-started/automatic-import.asciidoc

```diff
@@ -35,20 +35,19 @@ IMPORTANT: Using Automatic Import allows users to create new third-party data in
 image::images/auto-import-create-new-integration-button.png[The Integrations page with the Create new integration button highlighted]
 +
 3. Click **Create integration**.
-4. Select an <<assistant-connect-to-bedrock, Amazon Bedrock connector>>. 
+4. Select an <<llm-connector-guides, LLM connector>>. 
 5. Define how your new integration will appear on the Integrations page by providing a **Title**, **Description**, and **Logo**.  Click **Next**.
 6. Define your integration's package name, which will prefix the imported event fields. 
 7. Define your **Data stream title**, **Data stream description**, and **Data stream name**. These fields appear on the integration's configuration page to help identify the data stream it writes to.
 8. Select your {filebeat-ref}/configuration-filebeat-options.html[**Data collection method**]. This determines how your new integration will ingest the data (for example, from an S3 bucket, an HTTP endpoint, or a file stream).
-9. Upload a sample of your data in JSON or NDJSON format. Make sure to include all the types of events that you want the new integration to handle. 
+9. Upload a sample of your data. Make sure to include all the types of events that you want the new integration to handle. 
 +
 .Best practices for sample data
 [sidebar]
 --
-- The file extension (`.JSON` or `.NDJSON`) must match the file format.
-- Only the first 10 events in the sample are analyzed. In this technical preview, additional data is truncated. 
-- Ensure each JSON or NDJSON object represents an event, and avoid deeply nested object structures. 
-- The more variety in your sample, the more accurate the pipeline will be (for example, include 10 unique log entries instead of the same type of entry 10 times).
+- The file extension should match the file format.
+- Each object in your sample should represent an event, and you should avoid deeply nested object structures. 
+- The more variety in your sample, the more accurate the pipeline will be. Include a wide range of unique log entries instead of just repeating the same type of entry. Automatic Import will select up to 100 different events from your sample to use as the basis for the new integration. 
```

Comment:
@ilyannn do you think it's worth providing a brief description of how sampling works from https://github.com/elastic/kibana/pull/191598? Or at least clarify we're deterministically sampling the best 100 events and not just truncating (like before)?